specification/integration_context.md

```diff
@@ -0,0 +1,91 @@
+<!-- markdownlint-configure-file 
+{ 
+  "MD013": { 
+    "tables": false 
+  } 
+}
+-->
+
+# Integration Context
+
+**Status**: [Experimental](../README.md#versioning-and-status-of-the-specification)
+
+In order to create a more complete picture of a distributed environment, we
+propose the use of additional headers to pass extra context details between
+Cisco observability components. Specifically, we are concerned with passing
+information between OpenTelemetry-based Splunk distributions and AppDynamics
+agents, in both directions.
+
+# Headers
+
+The following defines several new headers and describes their intended use.
+All headers MUST be treated as optional -- peer services will not always
+generate them. These headers SHOULD be treated as opaque values of type
+string.
+
+* `cisco-ctx-acct-id` - Contains the ID of the AppDynamics account.
```

Comment:
Maybe `AppD` prefix as it feels more unique than `cisco`?

Reply:
What I like about `cisco` is that we are able to use the same prefix for data generated in AppD components and data generated in splunk-otel components. With this approach, the intention is to be able to say "Oh, I see this `cisco-ctx-*` header, I know exactly why it exists -- to facilitate integrated product experience.